libs/designsystem/src/lib/scss/_global-styles.scss

```
@@ -94,7 +94,7 @@ ion-modal.kirby-overlay {
       }
     }
 
-    &:not(.kirby-modal-compact).full-height {
+    &:not(.kirby-modal-compact).kirby-modal--full-height {
```

Please align with existing naming:
```suggestion
    &:not(.kirby-modal-compact).kirby-modal-full-height {
```